src/plone/restapi/tests/test_types.py

```diff
@@ -130,14 +130,14 @@ def test_get_jsonschema_for_portal_type(self):
         )
         self.assertEqual(jsonschema['title'], 'Page')
         self.assertEqual(jsonschema['type'], 'object')
-        self.assertIn('title', jsonschema['properties'].keys())
+        self.assertIn('title', list(jsonschema['properties'].keys()))
         self.assertIn('title', jsonschema['required'])
         self.assertEquals('default', jsonschema['fieldsets'][0]['id'])
         self.assertIn('title', jsonschema['fieldsets'][0]['fields'])
 
         jsonschema = get_jsonschema_for_portal_type(
             'Document', portal, request, excluded_fields=['title'])
-        self.assertNotIn('title', jsonschema['properties'].keys())
+        self.assertNotIn('title', list(jsonschema['properties'].keys()))
```

Comment:
this should be written: `self.assertNotIn('title', list(jsonschema['properties']))`